build.sbt

```diff
@@ -126,6 +127,8 @@ lazy val sqs = alpakkaProject("sqs",
 
 lazy val sse = alpakkaProject("sse", Dependencies.Sse)
 
+lazy val unixdomainsocket = alpakkaProject("unix-domain-socket", Dependencies.UnixDomainSocket)
+
```

Comment:
Build is currently failing due to 2.4 dependency. I'm using 2.5 features including CoordindatedShutdown and some newer features of Source. Is it possible for Alpakka components to exclude 2.4 support? I'd rather not invest the time to support 2.4 if possible.

Reply:
This problem is now rectified given https://github.com/akka/alpakka/pull/633